data/src/main/kotlin/hr/asee/android/template/data/model/remote/body/ApiRegisterRequest.kt

```diff
@@ -0,0 +1,10 @@
+package hr.asee.android.template.data.model.remote.body
+
+import com.google.gson.annotations.SerializedName
+
+data class ApiRegisterRequest (
+    @SerializedName("name") val name: String,
+    @SerializedName("email") val email: String,
+    @SerializedName("password") val password: String,
+    @SerializedName("confirmpassword") val confirmpassword: String
```

Comment:
Ova `@SerializedName` anotacija služi ako je neko polje u JSON-u koje backend očekuje kao request ili šalje kao response drugačije nego naziv varijable, u ovom slučaju bi možda to jedino bilo `confirmpassword` jer bi se varijabla idealno zval confirmPassword (camelCase opet)